docs/langref/relay_expr.rst

```diff
@@ -213,13 +214,14 @@ The below definition is also polymorphic, but restricts its
 arguments to tensor types:
 
 .. code-block:: python
+
     fn<s : Shape, bt : BaseType>(%x : Tensor[s, bt]) {
         %x
     }
 
 Notice that the return type is omitted and will be inferred.
 
-*Note: :code:`where` syntax is not yet supported in the text format.*
+*Note: "where" syntax is not yet supported in the text format.*
```

Comment:
It seems the RST format can't handle nested styles (i.e., a code block within an italic block).  If there are any alternatives to the fix I applied, let me know.